Misc/NEWS.d/next/IDLE/2019-12-05-14-20-53.bpo-38439.j_L2PI.rst

```diff
@@ -0,0 +1 @@
+Add a 256×256 pixel IDLE icon to support more modern environments.
```

Comment:
Mention maybe that Andrew Clover drew them.

Reply:
Sure.